src/pcb/pcb_plated_hole.ts

```diff
@@ -14,6 +15,20 @@ export const pcb_plated_hole = z
     pcb_component_id: z.string().optional(),
     pcb_port_id: z.string().optional(),
   })
+  .object({
+    type: z.literal("pcb_plated_hole"),
+    shape: z.literal("oval"),
+    outer_width: z.number(),
+    outer_height: z.number(),
+    hole_width: z.number(),
+    hole_height: z.number(),
+    x: distance,
+    y: distance,
+    layers: z.array(layer_ref),
+    port_hints: z.array(z.string()).optional(),
+    pcb_component_id: z.string().optional(),
+    pcb_port_id: z.string().optional(),
+  })
```

Comment:
This syntax doesnt seem right (i wish this module had tests), you probably need to use .or(

Reply:
Ohh yess!